testtools/files.go

```diff
@@ -260,6 +260,7 @@ func TestGazelleGenerationOnPath(t *testing.T, args *TestGazelleGenerationArgs)
 		})
 
 		testdataDir, cleanup := CreateFiles(t, inputs)
+		workspaceRoot := filepath.Join(testdataDir, args.Name)
```

Comment:
Alternatively, I can declare `redactWorkspacePath := func(s string) string` here, and have it captures the `workspaceRoot` to make call sites clearer. But this function is too damn long... I don't want to make it even more complex.

Reply:
Yes, this function is already too complex for an inlined anonymous. A refactor would be in order but I think is for another day.